core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -108,8 +108,9 @@ oppia.directive('feedbackPopup', [
 
           $scope.saveFeedback = function() {
             if ($scope.feedbackText) {
+              feedbackSubject = $scope.feedbackText.split(' ', 7).join(' '); 
```

Comment:
This might be an issue with languages that don't have spaces and are character-based. Perhaps use a character count instead.

Reply:
and you'll need to declare a var here :)

Reply:
Done